programmers/수박수박수박수박수박수?.js

```diff
@@ -0,0 +1,8 @@
+function solution(n) {
+    let answer = ""
+        for( let i = 1; i < n+1; i++){
+            if( i % 2 === 0) answer += '박';
+            else answer += '수';
+        }
+    return answer;
+}
```

Comment:
세미콜론 안적는거랑 for문 이거 들여쓰기 절대 절대 해선 안됨!
```suggestion
function solution(n) {
    let answer = ""; //세미콜론 꼭 추가해주기.
    
    //들여쓰기와 공백 더더 신경쓰기
    for (let i = 0; i < n; i++) {
        if (i % 2) answer += '박';
        else answer += '수';
    }
    return answer;
}
```

for문에서 i를 1부터 시작한 이유는 알겠는데 이런식으로 하면 좀 더 간결하게 코드 작성이 가능함.
i % 2 가 0이면 0은 boolean에서 false를 의미, 1이면 boolean에서 true를 의미

```js
function solution(n) {
    let answer = ""; 
    
    for (let i = 0; i < n; i++) {
        if (i % 2) answer += '박';
        else answer += '수';
    }
    return answer;
}
```